### ISSUE TITLE
cannot open (job sysout file)

### ISSUE THREAD

**Issue by cds-ari** [zowe/vscode-extension-for-zowe#442]
I can expand a JOB's separate sysout files in the JOBS explorer window but when I click on one I get the following (names and URL changed for privacy):

cannot open zosspool:JOBABCD.JOB00058.JESMSGLG%3F%5B%22zowprof1%22%2C%7B%22recfm%22%3A%22UA%22%2C%22records-url%22%3A%22https%3A%2F%2F***..*.98%3A10443%2Fzosmf%2Frestjobs%2Fjobs%2FJ0000058SVSCJES2D74B1E0E.......%3A%2Ffiles%2F2%2Frecords%22%2C%22stepname%22%3A%22JES2%22%2C%22subsystem%22%3A%22JES2%22%2C%22job-correlator%22%3A%22J0000058SVSCJES2D74B1E0E.......%3A%22%2C%22byte-count%22%3A1049%2C%22lrecl%22%3A133%2C%22jobid%22%3A%22JOB00058%22%2C%22ddname%22%3A%22JESMSGLG%22%2C%22id%22%3A2%2C%22record-count%22%3A18%2C%22class%22%3A%22X%22%2C%22jobname%22%3A%22JOBABCD%22%2C%22procstep%22%3Anull%7D%5D. Detail: Unexpected end of JSON input

**Comment by zFernand0** [comment]
The decoded object seems OK to me. (assuming we are the ones prepending `zosspool:JOBABCD.JOB00058.JESMSGLG?`)
```
zosspool:JOBABCD.JOB00058.JESMSGLG?["zowprof1",{"recfm":"UA","records-url":"https://***..*.98:10443/zosmf/restjobs/jobs/J0000058SVSCJES2D74B1E0E.......:/files/2/records","stepname":"JES2","subsystem":"JES2","job-correlator":"J0000058SVSCJES2D74B1E0E.......:","byte-count":1049,"lrecl":133,"jobid":"JOB00058","ddname":"JESMSGLG","id":2,"record-count":18,"class":"X","jobname":"JOBABCD","procstep":null}]
```

Could this be something non-encoded character in the shortened fields?

**Comment by cds-ari** [comment]
Thanks for replying. Forgive me but I do not understand your question. Can you please restate?

**Comment by zFernand0** [comment]
Absolutely.
I noticed some things had an ellipsis on the provided output. (e.g. `"job-correlator":"J0000058SVSCJES2D74B1E0E.......:"`).
So I was just wondering if this is the extension truncating the error message, or if this is the full output from the object?

And my question was that, in the event of some truncation, was there any badly encoded character in the truncated parts that we failed to handle?

**Comment by cds-ari** [comment]
I copied and paste the error message text as is and only changed the private info from one character string to another. I change the alphanumeric jobname to JOBABCD and I garbled the URL (IPA). Everything else remains as I originally copied it from the error message detail.

**Comment by jellypuno** [comment]
@cds-ari are you still experiencing this error?

**Comment by cds-ari** [comment]
Hi Jessielaine,

 

Last time I tried opening SYSOUT it worked. I have no idea how to recreate the error, if there still is one. Thanks for asking.

 

Thank you,

Ari Mor
Manager, Support Services
CAN DO SYSTEMS, INC.
 <http://www.candosys.com/> http://www.candosys.com/

This email may contain confidential and privileged material for the sole use
of the intended recipients. Any review, use, distribution or disclosure by
others is strictly prohibited. If you are not the intended recipient (or authorized
to receive for the recipient), please contact the sender by reply email and delete
all copies of this message. To reply to our email administrator directly, send an
email to  <mailto:webmaster@candosys.com?subject=Unintended%20recipient%20of%20Can%20Do%20Systems%20email> webmaster@candosys.com.

**Comment by zFernand0** [comment]
We may have fixed it in between 1.1.0 and 1.3.0 :smile:
Should we close this issue?

**Comment by jellypuno** [comment]
cool! thanks!
